src/orange/Data.hh

```diff
@@ -29,10 +29,15 @@ struct OrangeParamsScalars
     size_type max_intersections{};
     size_type max_logic_depth{};
 
+    // Multiplicative and additive values for bumping particles
+    real_type bump_rel{1e-8};
+    real_type bump_abs{1e-8};
```

Comment:
Yes actually, in SCALE we have to dial down the "tolerances" for surface merging because analysts like to put tiny gaps (0.001 cm or whatever) between outer surfaces in one universe and the inner surface in the next. In certain situations where we can't "remove" the daughter enclosing surface, this can leave a real gap between the boundaries across universes. To survive the crossing of these gaps, we have to bump a distance at least as much as the merge tolerance.

The relative vs absolute thing is because even though reactor people tend to work at a 1cm scale, there *will* be people doing DNA damage models in cells, and we don't want a hard absolute bump of 1e-6 cm to push us right across a cell.